### PR TITLE
Upgrade python requirement from 3.7 to 3.8, add clusterfuzz pkg

### PR DESCRIPTION
**Pull request by inferno-chromium**
- Upgrade python 3.7 to 3.8 in images.
- Upgrade pylint, pytype and yapf packages, fix lint failures.
- Add libClusterFuzz for stacktrace parsing.

### CODE REVIEW

**Review by lszekeres — APPROVED** [review]

**Comment by inferno-chromium** [comment]
Removed libClusterFuzz code since Oliver is putting a pip package, so that we can then just do pip install clusterfuzz. Also, this was kind of unrelated change to this 3.8 upgrade, so nice to go in another cl.

File: docker/dispatcher-image/Dockerfile [hunk 1]
```diff
@@ -41,6 +39,9 @@ RUN curl https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64 > \
     /usr/local/bin/cloud_sql_proxy
 RUN chmod +x /usr/local/bin/cloud_sql_proxy
 
+# Add libClusterFuzz for stacktrace parsing.
+RUN pip3 install clusterfuzz
```
Comment (jonathanmetzman) [inline docker/dispatcher-image/Dockerfile:43]: Why can't we put this in requirements.txt?
Linting will explode if we start using this package without doing that.

**Review by jonathanmetzman — COMMENTED** [review]
LGTM
Thanks!

File: docker/dispatcher-image/Dockerfile (hunk 1 above)
Comment (inferno-chromium) [reply docker/dispatcher-image/Dockerfile:43]: Sure, will do.

**Pull request merged**